src/main/java/pl/gov/coi/pomocua/ads/legalassistances/LegalAssistanceOffer.java

```diff
@@ -0,0 +1,46 @@
+package pl.gov.coi.pomocua.ads.legalassistances;
+
+import lombok.EqualsAndHashCode;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import pl.gov.coi.pomocua.ads.BaseOffer;
+import pl.gov.coi.pomocua.ads.Location;
+
+import javax.persistence.*;
+
+import java.util.List;
+
+import static javax.persistence.EnumType.STRING;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+public class LegalAssistanceOffer extends BaseOffer {
+    @Enumerated(STRING)
+    public Mode mode;
+
+    public Location location;
```

Comment:
JPA specifies, that when `@Embeddable` class is used (`Location` in this example), it should be marked as `@Embedded` in this place (see other entity classes for example)

Reply:
I see, changed. Thanks :)